src/components/RepoList.vue

```diff
@@ -1,15 +1,48 @@
 <template>
   <div id="repoList">
     <h1>{{ userName }}</h1>
+    <p v-show="!isEmptyRepoList">
+      레포지토리가 존재하지 않습니다.
+    </p>
+    <p v-show="isEmptyRepoList">
+      {{ repoNameList }}
+    </p>
   </div>
 </template>
 
 <script>
 export default {
   name: 'RepoList',
+  props: {
+    userName: String,
+  },
+  data() {
+    return {
+      isFetchSuccess: null,
+      repoNameList: [],
+    };
+  },
+  updated() {
+    this.fecthRepoList();
+  },
   computed: {
-    userName() {
-      return this.$store.state.userName;
+    isEmptyRepoList() {
+      return this.isFetchSuccess;
+    },
+  },
+  methods: {
+    fecthRepoList() {
+      this.$axios.get(`https://api.github.com/users/${this.userName}/repos`)
+        .then((res) => {
+          this.isFetchSuccess = true;
+          this.repoNameList = [];
+          res.data.forEach((element) => {
+            this.repoNameList.push(element.name);
+          });
+        })
```

Comment:
제가 vue는 잘 모르지만 아래는 어떠나유?! ㅎㅎ 
```suggestion
        .then(({data: repos}) => {
          this.isFetchSuccess = true;
          this.repoNameList = repos.map(repo => repo.name);
        })
```